### PR TITLE
remove unnecessary json parsing

### PR DESCRIPTION
**Pull request by dmfenton**
This PR changes DB select queries to request JSON instead of casting to string and then parsing. It would be nice to have some benchmarks to see how much this helps, but that's another project.

### CODE REVIEW

**Comment by ungoldman** [comment]
I've been seeing this off-by-one error pretty regularly with the pgcache travis tests, which will usually disappear after rerunning tests.

screen shot 2015-08-22 at 4 50 07 pm

@dmfenton any idea what might be causing it?

**Comment by dmfenton** [comment]
Fixed that problem in:
https://github.com/koopjs/koop-pgcache/pull/30

**Comment by ungoldman** [comment]
@dmfenton you know how to do the rebase magic?

**Comment by dmfenton** [comment]
rebased

File: index.js [hunk 1]
```diff
@@ -347,9 +347,9 @@ module.exports = {
         var info = result.rows[0].info
         var select
         if (options.simplify) {
-          select = 'select id, feature->>\'properties\' as props, st_asgeojson(ST_SimplifyPreserveTopology(ST_GeomFromGeoJSON(feature->>\'geometry\'), ' + options.simplify + ')) as geom from "' + id + ':' + (options.layer || 0) + '"'
```
Comment (ungoldman) [inline index.js:350]: Is the extra `>` changing the type? Just curious as a bit of a sql ignoramus

Comment (dmfenton) [inline index.js:350]: ->> means select as text -> means select the actual object
postgresql__documentation__9_3__json_functions_and_operators

http://www.postgresql.org/docs/9.3/static/functions-json.html

Comment (ungoldman) [inline index.js:350]: @dmfenton +1 thanks for the lesson

**Pull request merged**